src/components/GenQRcode.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+<div>
+  <canvas id="canvas"></canvas>
+</div>
+</template>
+<script>
+import QRCode from 'qrcode'
+export default {
+  props: {
+    qrcode: [String, Object],
+    color: {
+      type: Array,
+      default: () => ['#000', '#0000'],
+    },
+    size: {
+      type: Number,
+      default: () => 4,
+    },
+  },
+  mounted() {
+    var canvas = this.$el.querySelector('#canvas')
+    QRCode.toCanvas(
+      canvas,
+      this.qrcode,
+      {
+        color: {
+          dark: this.color[0], 
+          light: this.color[1],
+        },
+        version: this.size,
```

Comment:
version: this.size  , --> this means ? version is size ?

Reply:
For version 1 they are 21x21, for version 2 25x25 e so on. Higher is the version, more are the storable data, and of course bigger will be the QR Code symbol.